### ISSUE TITLE
messages.google.com text box that shouldn't be hidden is getting hidden

### ISSUE THREAD

**Issue by RedDragonWebDesign** [yourduskquibbles/webannoyances#523]
### URL(s) where the issue occurs

* https://messages.google.com/web/u/2/conversations/new

### Describe the issue

* This text box is getting incorrectly hidden:
![image](https://github.com/user-attachments/assets/dc77cca4-9f51-42bf-886e-80581e82f55d)

### Screenshot(s)

[Screenshot(s) for difficult to describe visual issues are **mandatory**. Post links instead of **Inline Images** for Screenshots containing **Adult material**.]

### Versions

- Browser & Version: [here]
- Adblocker (uBlock Origin, Adguard, Nano Adblocker, etc.) & Version: [here]

### Settings

- [List here all the changes you made to your Adblocker's default settings]

### Notes

* this filter is bad and needs removing/rewriting: `google.com#@##js-header, #searchform, .O-j-k, .header:not(#menu.header--primary):not(.column-section):not(.viewed):not(.ng-star-inserted):not([class*="_ngcontent"]):not([role="heading"]), .js-header.chr-header, .mobile-action-bar, body > .ng-scope, #flt-nav, .gws-flights__scrollbar-padding.gws-flights__selection-bar, .gws-flights__selection-bar-shadow-mask:style(position: absolute !important;)`

**Comment by Sonic5435325** [comment]
I don't think this filter is on our fork: https://github.com/LanikSJ/webannoyances
Let me know if you have any issues
